tests/ui/js/base-page-storyEditTests.js

```diff
@@ -863,6 +879,303 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: "Not really a URL"
+            },                  blockContentValues: "default",  expectedEmpty: false
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "An actual heading",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "Some alternative text",
+                description: "",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "A real description",
+                transcript: "",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "A real transcript",
+                mediaUrl: ""
+            },                  blockContentValues: "default",  expectedEmpty: true
+        },
+        {
+            block: {
+                blockType: "video",
+                heading: "",
+                alternativeText: "",
+                description: "",
+                transcript: "",
+                mediaUrl: "Not really a URL"
+            },                  blockContentValues: "default",  expectedEmpty: false
+        }
+    ];
+
+    sjrk.storyTelling.base.page.storyEditTester.verifyIsEmptyBlock = function (blockContentValues) {
+        fluid.each(isEmptyBlockTestCases, function (testCase, index) {
+            var blockContentValuesToUse = testCase.blockContentValues === "default" ? blockContentValues : testCase.blockContentValues;
```

Comment:
Instead of having the `"default"` flag, perhaps you can just test if the a value is provided and if not, use the default.